### DISCUSSION TITLE
I 'm Currenly get started with Data Science/ ML/DL

### DISCUSSION THREAD

**Discussion by CODGhost911** [Saisha16/DSA-Source-Codes#6] [General]
# What would a proper roadmap for that?

**Comment by Atharvkote** [comment]
## I Can Guide you with that !

### Python For DataScience RoadMap

1.Learn Python 
(Can Do DSA In it that would be more beneficial) 
2.Start Preparing Mathematics Concept side by side ( Statistics ,Linear Algebra, Probability )
3.Leran NumPy 
4.Learn Pandas
5.Learn Data Visualization Tool
Mathpo.lib
SeaBorn,etc
6.Starting Digging in BigData Handling 
7.learn Data Science concept 
Data cleaning 
Data Wrapping etc.

Hope it helped🧡💥💫

**Comment by CODGhost911** [comment]
Thanks mate😄
